packages/web3-utils/types/test.ts

```diff
@@ -0,0 +1,1116 @@
+/*
+    This file is part of web3.js.
+    web3.js is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file test.ts - please do not worry about the red underlines, we need to test for bad types
+ * so we are passing in bad types on purpose
+ * @author Josh Stevens <joshstevens19@hotmail.co.uk>
+ * @date 2018
+ */
+
+import {
+    isBN,
+    BN,
+    isBigNumber,
+    toBN,
+    toTwosComplement,
+    isAddress,
+    isHex,
+    isHexStrict,
+    asciiToHex,
+    hexToAscii,
+    toAscii,
+    bytesToHex,
+    numberToHex,
+    checkAddressChecksum,
+    fromAscii,
+    fromDecimal,
+    fromUtf8,
+    fromWei,
+    hexToBytes,
+    hexToNumber,
+    hexToNumberString,
+    hexToString,
+    hexToUtf8,
+    keccak256,
+    padLeft,
+    leftPad,
+    rightPad,
+    padRight,
+    sha3,
+    randomHex,
+    utf8ToHex,
+    stringToHex,
+    toChecksumAddress,
+    toDecimal,
+    toHex,
+    toUtf8,
+    toWei,
+    isBloom,
+    isTopic,
+    _fireError,
+    _jsonInterfaceMethodToString,
+    _flattenTypes,
+    soliditySha3,
+    getUnitValue,
+    unitMap,
+    testAddress,
+    testTopic
+} from 'utils';
+
+const bigNumber = new BN(3);
+
+// isBN Tests
```

Comment:
yeah was thinking that at the start but then i thought it may sit better in 1 file, can split it out. Will make sure the `dtslint` can run in that way before i do it though 👍

Reply:
👍